cabi/src/core.rs

```diff
@@ -91,6 +91,9 @@ pub enum SymbolicErrorCode {
     MissingSection = 1005,
     BadDwarfData = 1006,
     MissingDebugInfo = 1007,
+    BadJson = 1008,
```

Comment:
We should offset those and structure them more clearly, so we don't conflict that often. I'm using `2xxx` for the minidump stuff in #6. Minidump errors for illustration:

```rust
    // Rust standard errors
    NoError = 0,
    Panic = 1,
    Internal = 2,
    Msg = 3,
    Unknown = 4,
    Parse = 101,
    // Generic symbolic errors
    NotFound = 102,
    Format = 103,
    // Debug symbol processing errors
    BadSymbol = 1001,
    UnsupportedObjectFile = 1002,
    MalformedObjectFile = 1003,
    BadCacheFile = 1004,
    MissingSection = 1005,
    BadDwarfData = 1006,
    MissingDebugInfo = 1007,
    // Minidump errors
    Stackwalk = 2001,
    Resolver = 2002, // to be removed
    // Sourcemap errors
    BadJson = 3001,
    BadSourcemap = 3002,
    CannotFlattenSourcemap = 3003,
    // External crate errors
    Io = 10001,
    Utf8Error = 10002,
    ParseInt = 10003,
```